tvtk/vtk_parser.py

```diff
@@ -629,6 +629,437 @@ def _find_get_set_methods(self, klass, methods):
             elif (klass_name == 'vtkContextMouseEvent'
                   and method[3:] == 'Interactor'):
                 pass
+            elif (klass_name == 'vtkAbstractArray'
+                  and method[3:] == 'NumberOfComponents'):
+                continue
```

Comment:
All of these are get/set-able things that should return numeric values, but give `None` when queried. There are a handful of these in `master`, but a ton in VTK9 (this is what I'm only halfway through).

Reply:
Actually whether or not these work seems to depend on which classes I've instantiated, so these are probably/possibly related to memory corruption elsewhere

Reply:
> Actually whether or not these work seems to depend on which classes I've instantiated, so these are probably/possibly related to memory corruption elsewhere

Agreed. Many of seem to work just fine for me.